src/stars.css

```diff
@@ -0,0 +1,91 @@
+.tutto {
+  height: 160px;
+  width: 140px;
+}
```

Comment:
What is this tutto class, I believe you do not have it in your HTML

Reply:
tutto is Italian means "everything" or "all" ... 
it doesn't exist in the new version of stars.css
new version of stars.css is actually a masterpiece and Ihor accepted my stars.css